### PR TITLE
Update README and config

### LINKED ISSUE #10: Update README to be user-facing (remove instructions)
Remove customization and development instructions from the README.

Link to `<url for repository's GitHub Pages>/workshop/HOME` in the README, as this is where we want most users to start to interact with the repository.

For an example of a previous README, see here: <https://github.com/AlexsLemonade/2023-january-training/blob/main/README.md>

### PR DESCRIPTION
**Pull request by sjspielman**
~Closes #10~
Does most of #3 (will check off after merge)

This PR updates the README to be forward-facing, and fills in all config values except the docker image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,112 +1,3 @@
...
-### Running a local jekyll server
-
-Once all dependencies are installed, you should be able to start a local jekyll server with:
-
-```
-bundle exec jekyll serve
-```
+## 2024 June: Childhood Cancer Data Lab "Introduction to Single-Cell RNA-seq" Workshop
 
+Please find the workshop website at: https://alexslemonade.github.io/2024-june-training/
```
Comment (jashapiro) [inline README.md]: This is not active yet, so I might hold off on updating the readme?

Comment (sjspielman) [reply README.md]: I'll make sure the issue stays open. I'd like to get a draft schedule in before turning on pages I think?

Comment (jashapiro) [reply README.md]: > I'll make sure the issue stays open. I'd like to get a draft schedule in before turning on pages I think?

Makes sense to me

**Review by jashapiro — APPROVED** [review]
👔

**Pull request merged**